internal/core/user/uc-adaptor/ucclient/ucclient.go

```diff
@@ -26,11 +26,11 @@ import (
 	"github.com/sirupsen/logrus"
 
 	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/internal/core/user/uc"
```

Comment:
move uc-adaptor under uc dir.

Reply:
internal/core/user/
- impl/kratos/
- impl/uc/
  - impl/uc/uc-adaptor/

handler/ is strange.

Reply:
done